optimade/server/mappers/entries.py

```diff
@@ -34,10 +36,24 @@ def all_aliases(cls) -> Tuple[Tuple[str, str]]:
                 (f"_{CONFIG.provider.prefix}_{field}", field)
                 for field in CONFIG.provider_fields.get(cls.ENDPOINT, [])
             )
+            + tuple(
+                (f"_{CONFIG.provider.prefix}_{field}", field)
+                for field in cls.PROVIDER_FIELDS
+            )
```

Comment:
Why is this necessary? I thought this was included in the tuple defined just before, which comes from the config?

Reply:
I've answered this at your other comment below